_posts/2017-07-19-runners-driver.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Concordion | Runners and Drivers"
+heading: "Concordion Presentations"
+description: "Layering test code with Runners and Drivers"
+identifier: "concordion-presentations"
+author: "Nigel Charman"
+homepage: http://tutansblog.blogspot.co.nz/
+---
+
+![Visual representation of Layering test code with Runners and Drivers](_posts/drivers-runners.png)
```

Comment:
Please also move the image to the img folder and change the link to {{ site.baseurl }}/img/drivers-runners.png